app/hub/Views/SetupViews/SetupAntiSuiteView/SetupAntiSuiteView.scss

```diff
@@ -64,10 +64,10 @@
 }
 
 // Smart Browsing Icon
-.SetupAntiSuite__feature .smart-blocking {
+.SetupAntiSuite__feature .smart-browsing {
 	background-image: buildSmartBrowsingIcon(#9f9f9f);
 }
-.SetupAntiSuite__feature .smart-blocking.active {
+.SetupAntiSuite__feature .smart-browsing.active {
 	background-image: buildSmartBrowsingActiveIcon(#1dafed);
```

Comment:
Did you also update the place(s) in the components where these classes are used? Sorry if I missed that

Reply:
My bad! Looks like it got updated by the global search and replace 👍